CHANGELOG.md

```diff
@@ -5,40 +5,61 @@ All notable changes to Parcel will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and Parcel adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.10.1] – 2023-10-18
+
+### Fixed
+
+- Bundler
+  - Add logic to disable scope hoisting if the `this` key word is pointing to an export module [Details](https://github.com/parcel-bundler/parcel/pull/9291)
+  - updates the `build-bundles` script to use `rimraf` instead of `rm -rf` [Details](https://github.com/parcel-bundler/parcel/pull/9314)
```

Comment:
Can exclude this one from release notes since it is not user facing, just affected the mono repo package.json